scripts/brofish-ping.sh

```diff
@@ -27,14 +29,15 @@ brofish_ping() {
 
 brofish_cpu() {
   # Get CPU% from top
-  RESULT=$(top -bn1 -p$(cat /blog/current/.pid) |grep $BRO_PROC_NAME|awk '{print $9}')
-
-  if [[ ${RESULT%%.*} -ge $CPU_THRESHOLD ]]; then
-    echo ${RESULT%%.*}
-    return 1
-  else
-    return 0
-  fi
+  top -bn1|grep $BRO_PROC_NAME|awk '{print $1 " " $9}'|
+  while read PID CPU; do
```

Comment:
This "while" block sets the variables(PID CPU) with values read from pipe and processes them all inside its code block. So above "shopt -s lastpipe" is not needed

Reply:
removing `shopt -s lastpipe` will stop this script from working, probably because of the `return` statement below?

Reply:
This is strange and I am curious about the error, but it might take more time than necessary to fix it. Just go ahead with `shopt -s lastpipe` then.